x-pack/test/apm_api_integration/basic/tests/service_maps.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { JsonObject } from 'src/plugins/kibana_utils/common';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
Can we adjust the [ESLint exception for this](https://github.com/elastic/kibana/blob/ccf949cc39c1fccc0d78a4be02f649d8ecc03598/.eslintrc.js#L337)?

Reply:
Good idea 👍